rio/srcrobot/robot_container.py

```diff
@@ -199,23 +199,23 @@ def autoModeShot(self, autoShot: Constants.NextShot) -> Command:
         Used during automode commands to shoot any Constants.NextShot.
         """
         self.m_robotState.m_gameState.setNextShot(autoShot)
-        armTimeoutSec = (autoShot.m_armAngle / 45.0) + 1.0
+        shotTimeoutSec = (autoShot.m_armAngle / 45.0) + 1.0
         return SequentialCommandGroup(
             InstantCommand(
                 lambda: self.s_Shooter.setShooterVelocity(autoShot.m_shooterVelocity),
                 self.s_Shooter,
             ).alongWith(
-                self.s_Arm.servoArmToTarget(autoShot.m_armAngle).withTimeout(
-                    armTimeoutSec
+                self.s_Arm.servoArmToTarget(autoShot.m_armAngle),
+                WaitCommand(0.1)
```

Comment:
If it is a deadline group then we probably don't need the WaitCommand and instead just shoot when the shooter and arm are ready.